fdbserver/team/Serializer.h

```diff
@@ -0,0 +1,110 @@
+#ifndef FDBSERVER_TEAM_ENCODER_H
+#define FDBSERVER_TEAM_ENCODER_H
+
+#pragma once
+
+#include <cstdint>
+#include <cstring>
+#include <vector>
+
+#include "flow/crc32c.h"
+#include "flow/Error.h"
+#include "flow/error_definitions.h"
+#include "flow/serialize.h"
+
+/**
+ * Get the number of bytes of a serialized object. Only works with objects with POD variables.
```

Comment:
What does "POD" mean?

Reply:
POD is plain old data. Actually this is not quite accurate here. I updated the doc.